data/sh12a/generate_results.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# function convert{
+#         convertmc
+# }
+# set -o xtrace
+
+
+# basenames for 1-d plots
+bplot="NB_Z_narrow_dose NB_Z_narrow_dose_water NB_Z_narrow_LET NB_Z_narrow_LET_water NB_Z_narrow_QEFF"
```

Comment:
Why using `NB` everywhere ?

Reply:
it is very convenient for fast scripting to have a prefix for the resulting files, irrespective of suffix.

And I like the letters NB :)

Reply:
OK, thanks